### ISSUE TITLE
Apply Reverse line to selected features

### ISSUE THREAD

**Issue by jorgealmerio** [qgis/QGIS#37834]
It would be better if Reverse line tool could be apllied to all selected features.

When I want to change many I have to click on each, it's not good!

**Comment by andreasneumann** [comment]
I think you can already do that with processing and in place editing.Would that work for you?

I know it is 1-2 extra clicks, but it already works now.

**Comment by jorgealmerio** [comment]
Yes, It works! But as you said, "some extra clicks" and the tool button is already there in editing Bar. It could work with the same and better way as processing!
